x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/cold_start_duration.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FAAS_COLDSTART_DURATION } from '../../../../../common/elasticsearch_fieldnames';
+import { Setup } from '../../../../lib/helpers/setup_request';
+import { fetchAndTransformMetrics } from '../../fetch_and_transform_metrics';
+import { ChartBase } from '../../types';
+
+const chartBase: ChartBase = {
+  title: i18n.translate('xpack.apm.agentMetrics.serverless.coldStartDuration', {
+    defaultMessage: 'Cold start duration',
+  }),
+  key: 'cold_start_duration',
+  type: 'linemark',
+  yUnit: 'time',
```

Comment:
The current time unit is milliseconds. I'm not sure if / what time unit is needed / assumed if unit is set as `time`. Do we need to transform to microseconds or something like that?

Ignore, if this question does not make sense or can be resolved later in the UI code.

Reply:
not really, the UI is going to handle it.